server/models/user.js

```diff
@@ -3,7 +3,8 @@ export default (sequelize, DataTypes) => {
     username: {
       type: DataTypes.STRING,
       unique: true,
-      allowNull: false
+      allowNull: false,
+      primaryKey: true,
```

Comment:
`Profile model` references `User model` via username. I therefore make `User model's username field` a  primary key so that Sequelize association between the two models can work effectively.